p2p/conn/errors.go

```diff
@@ -0,0 +1,48 @@
+package conn
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrSendInvalidSecreteConnKey = errors.New("send invalid secret connection key")
+	ErrRecvInvalidSecreteConnKey = errors.New("received invalid secret connection key")
+	ErrChallengeVerification     = errors.New("challenge verification failed")
+	ErrChunkLength               = errors.New("chunk length is greater than dataMaxSize")
```

Comment:
I think it's better to switch to a custom error type `ErrChunkSize` to include additional context. It will also allow the maximum size to be dynamically adjusted.
`type ErrChunkSize struct {
	Received int
	Max int
}`